docs/_docs/guides/first-analytics-test.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Writing your first analytics test with Bobcat
+---
+
+This guide uses `bobcat-aem-junit` template to create tests. Some concepts are the same as in the [first test guide]({{site.baseurl}}/docs/guides/first-test/) and we won't explain them here.
+
+## Context
+
+We will write a simple Google Analytics test that will verify the datalayer implementation for the page load event.
+
+1. Open the Cognifide homepage.
+2. Check that the expected analytics datalayer is present.
+
+
+## Setup
+
+We used the gradle template so we have much of the job already done.
+All we need to to is to add the analytics module to our [runmode]({{site.baseurl}}/docs/modules/core/runmodes/) in default.yaml
```

Comment:
Shouldn't the dependency be added to build.gradle too?

Reply:
Fair point. The full setup description is in the docs/_docs/modules/analytics.md so I will just link this doc here